dictionaries/en.dictionary.itop.ui.php

```diff
@@ -719,6 +719,8 @@
 	'UI:CSVReport-Row-Issue-Reconciliation'     => 'failed to reconcile',
 	'UI:CSVReport-Row-Issue-Ambiguous'          => 'ambiguous reconciliation',
 	'UI:CSVReport-Row-Issue-Internal'           => 'Internal error: %1$s, %2$s',
+	'UI:CSVReport-Value-Issue-NoValue' 		=> 'No value',
+	'UI:CSVReport-Row-Issue-NbField'           => 'Not the expected number of fields (current : %1$s fields, expected :%2$s)',
```

Comment:
Small alignment issue:
```suggestion
	'UI:CSVReport-Row-Issue-Internal'           => 'Internal error: %1$s, %2$s',
	'UI:CSVReport-Value-Issue-NoValue'          => 'No value',
	'UI:CSVReport-Row-Issue-NbField'            => 'Not the expected number of fields (current : %1$s fields, expected :%2$s)',
```